tests/main/snap-service-stop-mode/task.yaml

```diff
@@ -18,6 +18,7 @@ execute: |
 
     echo "We can see it running"
     systemctl status snap.test-snapd-service.test-snapd-service|MATCH "running"
+    systemctl show -p MainPID snap.test-snapd-service.test-snapd-service > old-main.pid
```

Comment:
Need new tests to test the existing tests :)

Reply:
Heh :) I added the test -e for the files in the code to do exactly this ;)